apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-description-links/src/block-links-map.ts

```diff
@@ -0,0 +1,255 @@
+type BlockProps = {
+	url: string;
+};
+
+const blockLinks: Record< string, BlockProps > = {
+	/**
+	 * Core Blocks
+	 */
+	'core/template-part': {
+		url: 'https://wordpress.com/support/full-site-editing/theme-blocks/template-part-block/',
+	},
```

Comment:
Do these need to be objects?
```suggestion
	'core/template-part': 'https://wordpress.com/support/full-site-editing/theme-blocks/template-part-block/'
```

Reply:
Yeah, I don't know I did this originally... I think I had been working on it for too long and got tunnel vision haha.